lib/hammer_cli_katello/local_helper.rb

```diff
@@ -0,0 +1,10 @@
+module HammerCLIKatello
+  module LocalHelper
+
+    def local_only
+      return true if system("rpm -q foreman", :out => File::NULL)
+      fail "Export can only be run on the server and not remotely."
```

Comment:
I would change to

"This command can only be run on the same server that Foreman is running on, and cannot be run remotely".